### PR TITLE
manifest: order subsystems alphabetically

### PR DESCRIPTION
**Pull request by benediktibk**
Order the subsystems in the manifest file alphabetically, as they actually should be according to the comment within the manifest file.

Signed-off-by: Benedikt Schmidt <benedikt.schmidt@embedded-solutions.at>

### CODE REVIEW

**Review by galak — APPROVED** [review]

**Review by nashif — APPROVED** [review]

File: west.yml [hunk 1]
```diff
@@ -25,46 +25,59 @@ manifest:
...
+    - name: hal_espressif
+      west-commands: west/west-commands.yml
+      revision: 22e757632677e3579e6f20bb9955fffb2e1b3e1c
+      path: modules/hal/espressif
     - name: hal_infineon
       revision: f1fa8241f8786198ba41155413243de36ed878a5
       path: modules/hal/infineon
+    - name: lvgl
+      revision: 31acbaa36e9e74ab88ac81e3d21e7f1d00a71136
+      path: modules/lib/gui/lvgl
```
Comment (erwango) [inline west.yml:61]: lvgl should not be there

**Review by erwango — CHANGES_REQUESTED** [review]

**Review by erwango — APPROVED** [review]

**Comment by benediktibk** [comment]
Sorry for the multiple updates, I'm currently trying to get to know this ecosystem and learning how I can contribute something.

**Comment by galak** [comment]
> Sorry for the multiple updates, I'm currently trying to get to know this ecosystem and learning how I can contribute something.

No worries, thanks for contributing!

**Review by mbolivar-nordic — COMMENTED** [review]
This needs a rebase due to the following change:

```

-hal_nxp      modules/hal/nxp              73445ce7474ffb0eb6fd795eeaac1fbf507e4b7e https://github.com/zephyrproject-rtos/hal_nxp
+hal_nxp      modules/hal/nxp              0d11138724959e1162777d9206f841ccdf64348e https://github.com/zephyrproject-rtos/hal_nxp
```

**Comment by benediktibk** [comment]
Do I have to trigger the checks "Run compliance checks on patch series (PR)" and "Scan code for licenses" manually?

**Comment by galak** [comment]
> Do I have to trigger the checks "Run compliance checks on patch series (PR)" and "Scan code for licenses" manually?

I triggered that for you.  Github "feature" that for first time contributors we have to click a button to do it.

**Pull request merged**